downstream/modules/platform/proc-create-postresql-secret.adoc

```diff
@@ -0,0 +1,30 @@
+[id="create-postresql-secret_{context}"]
+
+= Creating a postgresql configuration secret
+
+[role=_abstract]
+
+For migration to be successful, you must provide access to the database for your existing deployment.
+
+.Procedure
+
+. Create a yaml file for your postgresql configuration secret:
++
+-----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <resourcename>-old-postgres-configuration
+  namespace: <target namespace>
+stringData:
+  host: "<external ip or url resolvable by the cluster>"
+  port: "<external port, this usually defaults to 5432>"
+  database: "<desired database name>"
+  username: "<username to connect as>"
+  password: "<password to connect with>"
+type: Opaque
+-----
```

Comment:
Should this be followed by "+" ? Perhaps this one doesn't. Unfortunately, I'm only looking at the previews per each file here in GitHub; haven't figured out yet how to download a build from your PR. Just wanted to check to ensure it builds properly.